tools/GlobalFilters.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="utf-8"?>
+<GlobalFilters xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <ContentSearchFilters>
+    <ContentFilter>
+      <Name>Initial ContentFilters</Name>
+      <Filters>
```

Comment:
I added `user ID` as a content filter. We had `user=User ID` in one of the tests and it was showing up as a match